common.yml

```diff
@@ -1412,6 +1404,23 @@ tasks:
 
 buildvariants:
 
+#######################################
+#     Static Analysis Buildvariant    #
+#######################################
+
+- name: static
+  display_name: '! Static Analysis'
```

Comment:
why is this "! Static Analysis" instead of just "Static Analysis"?

Reply:
I actually borrowed this trick from the server evergreen project: they just use it to keep certain variants at the top of the list in patches/waterfall